src/test/java/eu/dissco/backend/service/AnnotationServiceTest.java

```diff
@@ -105,7 +105,7 @@ void testGetAnnotationsForUser() throws Exception {
     var receivedResponse = service.getAnnotationsForUser(userId, pageNumber, pageSize, path);
```

Comment:
Maybe also remove response from `receivedResponse`?

Reply:
not sure what you mean here

Reply:
You remove `Response` from `expectedResponse` only change I see in this unit test. However, the received value is still called `receivedResponse`. Not sure why this change was needed but thought it would like better if it `Response` was removed consistently